script/openqa-clone-custom-git-refspec

```diff
@@ -39,4 +39,5 @@ build="${build:-"$repo_name#$pr"}"
 casedir="${casedir:-"$repo#$branch"}"
 GROUP="${GROUP:-0}"
 dry_run="${dry_run:-""}"
-$dry_run openqa-clone-job --skip-chained-deps --within-instance $host $job _GROUP=$GROUP TEST=$test BUILD=$build CASEDIR=$casedir PRODUCTDIR=$productdir NEEDLES_DIR=$needles_dir
+shift 2 # eat the two params so that $@ contains only test vars
+$dry_run openqa-clone-job --skip-chained-deps --within-instance $host $job _GROUP=$GROUP TEST=$test BUILD=$build CASEDIR=$casedir PRODUCTDIR=$productdir NEEDLES_DIR=$needles_dir $@
```

Comment:
Quote the `$@` like `"$@"` so a test variable's value can contain a white-space.

Reply:
done